altair/utils/headless.py

```diff
@@ -122,6 +118,8 @@ def spec_to_image_mimebundle(spec, format, mode,
     driver_timeout : int (optional)
         the number of seconds to wait for page load before raising an
         error (default: 10)
+    webdriver_class : Type[Union[webdriver.Chrome, webdriver.Firefox]]
+        Webdriver to use (default: webdriver.Chrome).
```

Comment:
I think it would be cleaner to let this argument be a string (``webdriver='chrome'`` or ``webdriver='firefox'``), and then import the associated webdriver class below.